cards/fields.py

```diff
@@ -24,3 +25,19 @@ def pre_save(self, model_instance, add: bool):
                 raise TypeError(f'{model_instance.card} is not a valid Card')
 
         return super().pre_save(model_instance, add)
+
+
+class CardsField(models.TextField):
```

Comment:
This is the new field. If/when we want to make each card unique, we'll need to update this guy. Should be pretty easy to do.